playwright/README.md

```diff
@@ -179,14 +182,21 @@ this.customTypesLink = this.menu.getByText("Custom types", {
 });
 ```
 
-3. Don't use `.locator()`
+### Prefer using `.toBeVisible()` when testing the presense of visble elements
+
+`.toBeVisible()` ensures the element is present and visible to users. Explicitly checking for visibility prevents a false-positive test where the element is in the DOM, but cannot be seen or interacted by the user.
+
+> [!NOTE]
+> This preference can be ignored if the targeted element is hidden.
+
+If the element is purposely hidden, use
```

Comment:
You miss something here, correct?

Reply:
Haha oops, thanks for catching that. I must have gotten distracted half way through the sentence. 😅